ui/app/templates/components/dashboard/replication-card.hbs

```diff
@@ -2,7 +2,7 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 ~}}
-<Hds::Card::Container @hasBorder={{true}} class="has-padding-l has-bottom-padding-m">
+<Hds::Card::Container @hasBorder={{true}} class="has-padding-l has-bottom-padding-m" data-test-replication-card>
```

Comment:
if this selector is something like `data-test-card="replication"` the same selector could be reused for your other cards in tests

```js
SELECTORS = {
...
card: (name) => `[data-test-card="${name}"]`
}

Reply:
Thanks for pointing this out and great idea! I will make a follow up ticket!